cmake/macros.cmake

```diff
@@ -0,0 +1,18 @@
+
+# Create a Python module using pybind11
+#
+# Args:
+#     module_name:  The name of the module.  This macro expects the
+#         implementation to be found in a file `srcpy/${module_name}.cpp`.
+#     ARGN:  Any additional arguments are added to `target_link_libraries` so
+#         use this for any additional libraries that are not already covered by
+#         `${catkin_LIBRARIES}`.
+macro(add_python_module module_name)
```

Comment:
I would say this name is too general. Something including pybind11 and catkin would be appreciated.

Reply:
Good point.  Not sure what's the best order to stack the names.  I went for `add_catkin_pybind11_module` now.

Reply:
All good, I used [this](https://github.com/machines-in-motion/mpi_cmake_modules/blob/master/cmake/pkg-config.cmake#L830) in mpi_cmake_module

Reply:
Not sure I understand your comment correctly, shall I rename it?  I'm also good with `catkin_add_pybind11_module`.

Reply:
I renamed it to `catkin_add_pybind11_module` now.

Reply:
Perfect good to merge! Sorry I was confusing